acos_client/v30/slb/service_group.py

```diff
@@ -88,9 +95,9 @@ def create(self, name, protocol=TCP, lb_method=ROUND_ROBIN, **kwargs):
         self._set(name, protocol, lb_method, **kwargs)
 
     def update(self, name, protocol=None, lb_method=None, health_monitor=None,
-               **kwargs):
+               health_monitor_disable=0, **kwargs):
```

Comment:
Realized I didn't answer your other question - it's called in the HealthMonitorHandler when the monitor is deleted or dissociated from it's pool.